### ISSUE TITLE
Swagger model contains all information from all classes that inherit a base model

### ISSUE THREAD

**Issue by Ahagan** [Romakita/ts-express-decorators#128]
## Information

Type | Version
---|---
Bug | 2.3.8

****

## Description
If a model class inherits a base class that also contains the swagger decorators (e.g. @Title('')) then when the swagger documentation is generated, the model contains details about all properties from all models that inherit from the base class.

E.g.

Base Model ->
```typescript
export abstract class BaseModel {
    @Title('id')
    @Description('Unique identifier.')
    @JsonProperty()
    public id?: string;
}
```
Models ->
```typescript
export class NameModel extends BaseModel{
    @Title('name')
    @Description('The name')
    @JsonProperty()
    public name: string;
}

export class AgeModel extends BaseModel{
    @Title('age')
    @Description('The age')
    @JsonProperty()
    public age: number;
}
```

output model 
- `id`: Unique identifier
- `name`: The name
- `age`: The age

## Todos

- [ ] Fix swagger documentation generation if a class inherits from a base class with the swagger decorators on them.

**Comment by Romakita** [comment]
@Ahagan Thanks for this issues :) I'll fixed that ASAP ;)

**Comment by Romakita** [comment]
@Ahagan  Fixed :) with 2.4.1
